bot/exts/help_channels/_cog.py

```diff
@@ -229,16 +232,26 @@ async def get_available_candidate(self) -> discord.TextChannel:
 
         try:
             channel = self.channel_queue.get_nowait()
+
+            time_since_last_notify_seconds = (arrow.utcnow() - self.last_running_low_notification).seconds
+            within_interval = time_since_last_notify_seconds >= self.notify_interval_seconds
+            if within_interval and self.channel_queue.qsize() <= constants.HelpChannels.notify_running_low_threshold:
+                await _message.notify_running_low(
+                    self.bot.get_channel(constants.HelpChannels.notify_channel),
+                    self.channel_queue.qsize()
+                )
+                self.last_running_low_notification = arrow.utcnow()
```

Comment:
I think it'd be better to put in the `else` of the try-except, since the exception being caught has nothing to do with this code.

Also, what do you think about defining a predicate for this in `_messages.py` to reduce the amount of code in the cog? I'm always concerned that the cog is growing too big and things need to be refactored out of it to stay manageable. Such predicate could probably be used by both this code and the code in the `except` block.

Reply:
You should add stat for this similar to `help.out_of_channel_alerts`.